components-2/readme.md

```diff
@@ -0,0 +1,166 @@
+    commit
+    commit
+    checkout NW6
+    branch feature/card
+    commit
+    commit
+    commit
+    checkout NW6
+    branch feature/menu
+    commit
+    commit
+    commit
+    checkout NW6
+    merge feature/header
+    merge feature/footer
+    merge feature/card
+    merge feature/menu
+```
+
+Check you are in the right place with GitLens!
+
+<!--{{</note>}}-->
+
+<!--{{<note type="tip" title="Your working branch is your cohort name">}}-->
+
+Your COHORT branch is your cohort name. It might be NW6, WM5, GL7...You're working in your COHORT branch and NOT in main.
+
+<!--{{</note>}}-->
+
+Next, divide the class into four teams. Use the [team generator](teams.html) to help you do this.
+
+Set a whole class [timer for 10 minutes](https://www.google.com/search?q=10+minute+time).
+
+### Each team, choose a driver
+
+1. Driver, ask everyone in your team to confirm you are in the right working branch.
+1. Driver, from this branch, create and checkout a new branch for your team named like this: `feature/component`, where _cohort_ is your cohort name and _component_ is the component you're building, for example: `feature/header`
+
+```bash
+git switch --create feature/cohort-component
+```
+
+Now you're all set up to make changes to together.html.
+
+<!--{{</note>}}-->
+
+<!--{{<note type="activity" title="Find your docs 10m">}}-->
+
+## Find your docs
+
+Set a whole class [timer for 10 minutes](https://www.google.com/search?q=10+minute+time).
+
+Deployed from this repo is a [mini design system](https://cyf-workshops.netlify.app/components-2). This design system has several documented components, and some also have javascript files.
+
+Your mission is to find the docs for each component with an associated script file, and read them. Discuss your findings. Look at the pages on the website with Devtools. Compare the Inspect view with the source code you can see in the files. How are they different? How are HTML and the DOM different?
+
+<!--{{</note>}}-->
+
+### Together separately
+
+Set a whole class [timer for 30 minutes](https://www.google.com/search?q=30+minute+time).
+
+<!--{{<note type="activity" title="Parallel Development 30m">}}-->
+
+Now **build** your component. Elect a new driver.
+
+1. Find [together.html](together.html) in the /components-2 directory. Your mission is to complete this page.
+1. In the correct portion of the page, make your changes to build _your feature_ on together.html.
+1. Driver, commit your changes to your feature branch.
+1. Open a pull request from your branch to the COHORT branch, with your changes. Ask for help!
+1. Remember, docs will help you too.
+1. Do the simplest thing that could possibly work to begin with. You can always update your PR with more changes.
+
+```mermaid
+gitGraph
+    commit id: "start"
+    branch feature/component
+    commit id: "import component function"
+    commit id: "import data"
+    commit id: "lighthouse revisions"
+    checkout COHORT
+    merge feature/component
+```
```

Comment:
I'm not a mermaid expert but this seems to be throwing an error?

Reply:
oh it should be 

```mermaid
gitGraph
    commit id: "start"
    branch FOR-USE-IN-CLASS
    branch COHORT
    branch feature/component
    commit id: "import component function"
    commit id: "import data"
    commit id: "lighthouse revisions"
    checkout COHORT
    merge feature/component
  ```
  
  Somehow I didn't notice this despite having the mermaid vscode installed! TY